source/frontend/src/components/common/UserNameTooltip.tsx

```diff
@@ -39,12 +39,7 @@ export const UserNameTooltip: React.FunctionComponent<
       toolTipId={'userNameTooltip'}
       innerClassName={'userNameTooltip'}
       toolTip={userNameInfo}
-      customToolTipIcon={<StyledUserLabel>{userName ?? 'USER'}</StyledUserLabel>}
+      customToolTipIcon={<strong>{userName ?? 'USER'}</strong>}
```

Comment:
this affects existing code I think, does the change to strong here keep the secondaryVariantColor?

Reply:
**TooltipIcon.scss** already defines that anything contained by it will have the secondary variant color, so it was redundant in the custom icon. The color is preserved.

```scss

.tooltip-icon {
  min-width: 1.4rem;
  min-height: 1.4rem;
  color: $secondary-variant-color; // <---
  margin: 0rem 0.2rem;
}

```